### PR TITLE
[Webhook] Add bridges for Webhook

### LINKED ISSUE #18692: [RemoteEvent][Webhook] Add Mailjet support
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#50502
| PR author(s) | @blaugueux
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18692, https://github.com/symfony/symfony-docs/issues/18694, https://github.com/symfony/symfony-docs/issues/18745, https://github.com/symfony/symfony-docs/issues/18800

### CODE REVIEW

File: webhook.rst [hunk 1]
```diff
@@ -84,8 +84,17 @@ Mailer service  Parser service name
 =============== ==========================================
 Mailgun         ``mailer.webhook.request_parser.mailgun``
 Postmark        ``mailer.webhook.request_parser.postmark``
+Mailjet         ``mailer.webhook.request_parser.mailjet``
+Brevo           ``mailer.webhook.request_parser.brevo``
+Vonage          ``mailer.webhook.request_parser.vonage``
+Sendgrid        ``mailer.webhook.request_parser.sendgrid``
```
Comment (OskarStark) [inline webhook.rst]: Can you please rebase, and add versionadded directives for 6.3 too?


Also Bravo, Vonage and Mallet needs to get a `yes` in the mailer.rst file in webhook column

File: webhook.rst [hunk 2]
```diff
@@ -84,13 +84,16 @@ Mailer service  Parser service name
 =============== ==========================================
 Mailgun         ``mailer.webhook.request_parser.mailgun``
 Postmark        ``mailer.webhook.request_parser.postmark``
+Mailjet         ``mailer.webhook.request_parser.mailjet``
+Brevo           ``mailer.webhook.request_parser.brevo``
+Vonage          ``mailer.webhook.request_parser.vonage``
 Sendgrid        ``mailer.webhook.request_parser.sendgrid``
```
Comment (OskarStark) [inline webhook.rst]: Alpha sort please

**Comment by TimoBakx** [comment]
Might the wiring be ~missing~ incorrect for these bridges? They seem to be missing here: https://github.com/symfony/symfony/blob/a3686fbfbbbfec317213f831eabeb5a450720545/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L2589C15-L2589C15

File: webhook.rst (hunk 1 above)
Comment (alexandre-daubois) [reply webhook.rst]: Vonage is not present in Mailer. But I added for the others

Comment (TimoBakx) [reply webhook.rst]: Vonage is part of the Notifier. Should we mention webhook support there as well?

**Review by TimoBakx — APPROVED** [review]

**Comment by alexandre-daubois** [comment]
Ok I think this should be put on hold and waiting code merge of https://github.com/symfony/symfony/pull/53004

Then we'll see if this goes in 7.1 or if it is considered as a bugfix that should target 6.4.

File: webhook.rst (hunk 1 above)
Comment (alexandre-daubois) [reply webhook.rst]: I'd say yes! I reworded the sentence above the table to remove "mailer". What do you think?

Comment (TimoBakx) [reply webhook.rst]: Sounds good, but then also remove/rename "Mailer" from the table header. ^_^

**Comment by TimoBakx** [comment]
I have a slightly different alternative for Twilio and Vonage, as they use a different event from the Mailer implementations:
https://github.com/symfony/symfony-docs/pull/19269

Vonage is added in 6.4, I can make a follow-up PR for that bit.

File: webhook.rst (hunk 1 above)
Comment (alexandre-daubois) [reply webhook.rst]: Removed :+1:

**Comment by alexandre-daubois** [comment]
Alright I think we can actually merge this as is. The PR I opened on the code repo only checks that we remove some ids from the container conditionally, but the ids already exist. :+1: 

Your follow is a great idea, Timo!

File: webhook.rst [hunk 3]
```diff
@@ -77,20 +77,27 @@ The Webhook component routing needs to be defined:
...
+
+======== ==========================================
+Service  Parser service name
+======== ==========================================
+Brevo    ``mailer.webhook.request_parser.brevo``
+Mailgun  ``mailer.webhook.request_parser.mailgun``
+Mailjet  ``mailer.webhook.request_parser.mailjet``
+Postmark ``mailer.webhook.request_parser.postmark``
+Sendgrid ``mailer.webhook.request_parser.sendgrid``
+Vonage   ``mailer.webhook.request_parser.vonage``
```
Comment (TimoBakx) [inline webhook.rst]: This service (Vonage) can be removed from this list. It's not a `mailer` service, but one for `notifier`.

Comment (alexandre-daubois) [reply webhook.rst]: I agree I have to update the service alias. But don't we want to keep all third party services supported by Webhook here?

Comment (TimoBakx) [reply webhook.rst]: I made two separate lists in my other PR. One for Mailer and one for Notifier.

Comment (TimoBakx) [reply webhook.rst]: We can also merge this PR as is, and then I'll fix it in the other PR. ^_^

Comment (alexandre-daubois) [reply webhook.rst]: Suits me fine! Good for you @OskarStark? :slightly_smiling_face:

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Comment by OskarStark** [comment]
Below 50 issues 💪

**Pull request merged**